test/e2e/data/infrastructure-vsphere/cluster-template.yaml

```diff
@@ -4,6 +4,8 @@ kind: Cluster
 metadata:
   name: '${ CLUSTER_NAME }'
   namespace: '${ NAMESPACE }'
+  labels:
+    cni: "${ CLUSTER_NAME }-crs-0"
```

Comment:
If we start installing CNIs,  the `PodCIDR: 192.168.0.0/16` will conflict with `192.168.8.0/24` which is what we're using for machines

Reply:
E2E is using `192.168.0.0/16` now, is this a recent change?

Reply:
@sedefsavas - sorry I missed this, it's not a recent all of our e2e VM subnets are located within that range.